sw/device/lib/dif/dif_entropy_src.c

```diff
@@ -48,6 +52,35 @@ static void set_config_register(const dif_entropy_src_t *entropy,
                       reg);
 }
 
+/**
+ * Sets the `entropy` source firmware override register with the settings
+ * derived from `config`.
+ */
+static dif_entropy_src_result_t fw_override_set(
```

Comment:
Hate to do this to you too, but you may want to rebase on #8412 as well. The entropy_src DIFs were updated in #8412 to use the global DIF return type `dif_result_t`. Sorry for the hassle on this!

(Applies throughout).